.php_cs.dist

```diff
@@ -3,17 +3,15 @@
 use PhpCsFixer\Config;
 use PhpCsFixer\Finder;
 
-$year = (new DateTime())->format('Y');
-
 $header = <<<EOF
-RatePAY PHP-Library
+Ratepay PHP-Library
 
 This document contains trade secret data which are the property of
 RatePAY GmbH, Berlin, Germany. Information contained herein must not be used,
 copied or disclosed in whole or part unless permitted in writing by RatePAY GmbH.
 All rights reserved by RatePAY GmbH.
```

Comment:
Shouldn't this be also changed to Ratepay?

Reply:
done